dev-packages/cli/src/theia.ts

```diff
@@ -49,107 +51,107 @@ function rebuildCommand(command: string, target: ApplicationProps.Target): yargs
             command: 'start',
-            describe: 'start the ' + manager.pck.target + ' backend',
-            handler: async () => {
-                try {
-                    manager.start(commandArgs('start'));
-                } catch (err) {
-                    console.error(err);
-                    process.exit(1);
-                }
+            describe: `Start the ${target} backend`,
+            handler: async argv => {
+                manager.start(getCommandArgv(argv._));
             }
         })
         .command({
             command: 'clean',
-            describe: 'clean for the ' + target + ' target',
-            handler: () => {
-                try {
-                    manager.clean();
-                } catch (err) {
-                    console.error(err);
-                    process.exit(1);
-                }
+            describe: `Clean for the ${target} target`,
+            handler: async () => {
+                await manager.clean();
             }
         })
         .command({
             command: 'copy',
-            handler: () => {
-                try {
-                    manager.copy();
-                } catch (err) {
-                    console.error(err);
-                    process.exit(1);
-                }
+            describe: 'Copy various files from `src-gen` to `lib`',
+            handler: async () => {
+                await manager.copy();
             }
         })
-        .command({
+        .command<{
+            'mode': 'development' | 'production',
+            'split-frontend'?: boolean
+        }>({
             command: 'generate',
-            handler: () => {
-                try {
-                    manager.generate();
-                } catch (err) {
-                    console.error(err);
-                    process.exit(1);
-                }
+            describe: `Generate various files for the ${target} target`,
+            builder: cli => ApplicationPackageManager.defineGeneratorOptions(cli),
+            handler: async argv => {
+                await manager.generate({
+                    mode: argv.mode,
+                    splitFrontend: argv['split-frontend']
+                });
             }
         })
-        .command({
+        .command<{
+            'mode': 'development' | 'production',
+            'split-frontend'?: boolean
+        }>({
             command: 'build',
-            describe: 'webpack the ' + target + ' frontend',
-            handler: async () => {
-                try {
-                    await manager.build(commandArgs('build'));
-                } catch (err) {
-                    console.error(err);
-                    process.exit(1);
-                }
+            describe: `Generate and bundle the ${target} frontend using webpack`,
+            builder: cli => ApplicationPackageManager.defineGeneratorOptions(cli),
+            handler: async argv => {
+                console.log('AAAAAAAAAA', argv._);
```

Comment:
This is probably some leftover from debugging?

Reply:
Ahah yes, oops.